server/plugin/plugin.go

```diff
@@ -218,6 +218,9 @@ func (p *Plugin) ServeHTTP(pc *plugin.Context, w http.ResponseWriter, req *http.
 
 func (p *Plugin) IsAuthorizedAdmin(mattermostUserID string) (bool, error) {
 	env := p.getEnv()
+	if mattermostUserID == env.BotUserID {
```

Comment:
This is weird, but makes sense

Reply:
It is indeed weird. Merge error bringing back the changes to use the bot account. Will fix.